### PR TITLE
Couple of important fixes

### PR DESCRIPTION
**Pull request by iain17**
I used your project to write a quick wordpress plugin. However using it I found a couple of mistakes which I ended up fixing. Awesome work btw, just wee changes.

### CODE REVIEW

**Comment by barryvdh** [comment]
Thanks! Not really sure why that exception is called like that, seems a bit unclear. Should we rename it (and don't forget to change the throws phpdoc ) ? 

Also,  can you exclude your idea folder?

**Comment by barryvdh** [comment]
I think we can just return null, without the exception actually..

File: src/Match.php [hunk 1]
```diff
@@ -197,4 +197,14 @@ class Match extends AbstractItem
...
     }
+
+    /**
+     * De datum + tijd in unix time stamp
+     * @return int
+     */
+    public function getTime() {
+        //Time is saved as a string formatted like: 1830.
+        $time = implode(":", str_split($this->Tijd, 2));
+        return strtotime($this->Datum . ' '.$time);
```
Comment (barryvdh) [inline src/Match.php:208]: Misschien netter om DateTime::createFromFormat() te gebruiken en DateTime object te maken?

**Comment by barryvdh** [comment]
And what is the link to your plugin, so I can add it to the Readme?

**Comment by barryvdh** [comment]
Pulled all but the Exception change. Changed to DateTime in 659e26d341f38218a586c605392911508514b1f1

Good like this?

**Comment by iain17** [comment]
Sorry for the late reply. My wordpress plugin can be found on: https://github.com/iain17/knvb-wordpress

The pull is good. I did however find another little mistake when requesting the results of a team. See: https://github.com/iain17/php-knvb-dataservice-api/commit/7d70aef2b2f6ac5e50d0152de54bc50f15050666

The KNVB api sometimes gives back a null value for the team scores. No mention of this on their documentation website however. Without this little fix, everything stops working and it only spits out this error: JSON property "PuntenTeam1" must not be NULL.

**Comment by barryvdh** [comment]
Can you submit a PR for that?

**Comment by barryvdh** [comment]
Thanks, pulled it here: https://github.com/fruitcake/php-knvb-dataservice-api/commit/ab69a2c36ac6c1fe6e1786b7c6bc3180adf2bedd

**Pull request closed without merging**